travis/pip.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Used to return the result code given by pip instance of by grep
+# Used to return the result code given by pip instead of grep
```

Comment:
I add this comment for a previous version that less comprehensible, I think that this comment can be removed.